plugins/adrv9009.c

```diff
@@ -261,8 +261,8 @@ static void multichip_sync()
 
 		guint n = 0;
```

Comment:
why not squash this into the previous patch ?

Reply:
Because it is @mirceacaprioru's work and findings.

Reply:
ok
fine from me